### PR TITLE
[prometheus-elasticsearch-exporter] allow configuration of deployment labels

### PR DESCRIPTION
**Pull request by FIIRLEBR**
#### What this PR does / why we need it:

Allows configuration of deployment labels.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-elasticsearch-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 4.12.0
+version: 4.12.1
```
Comment (monotek) [inline charts/prometheus-elasticsearch-exporter/Chart.yaml:4]: new feature shoudl raise minor version.
```suggestion
version: 4.13.0
```

**Review by monotek — CHANGES_REQUESTED** [review]
value should aslo be reflected in values.yaml

File: charts/prometheus-elasticsearch-exporter/templates/deployment.yaml [hunk 2]
```diff
@@ -7,6 +7,9 @@ metadata:
     app: {{ template "elasticsearch-exporter.name" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{- if .Values.deployment.labels }}
```
Comment (monotek) [inline charts/prometheus-elasticsearch-exporter/templates/deployment.yaml:10]: the variable should also be reflected in values.yaml

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
The rules are genrated by the files in the hack directory, which downloads them.
Please fix upstream.

File: charts/prometheus-elasticsearch-exporter/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 4.14.0
+version: 4.14.1
```
Comment (monotek) [inline charts/prometheus-elasticsearch-exporter/Chart.yaml:4]: New feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-elasticsearch-exporter/Chart.yaml (hunk 3 above)
Comment (zanhsieh) [reply charts/prometheus-elasticsearch-exporter/Chart.yaml:4]:
```suggestion
version: 4.15.0
```

**Review by monotek — CHANGES_REQUESTED** [review]
Please rebase to fix dco. You might need to squash the online commit.

**Review by zanhsieh — APPROVED** [review]

**Review by monotek — APPROVED** [review]

**Pull request merged**